aspnetcore/blazor/call-dotnet-from-javascript.md

```diff
@@ -19,6 +19,9 @@ This article covers invoking .NET methods from JavaScript. For information on ho
 
 [View or download sample code](https://github.com/dotnet/AspNetCore.Docs/tree/master/aspnetcore/blazor/common/samples/) ([how to download](xref:index#how-to-download-a-sample))
 
+> [!NOTE]
+> The order that custom JavaScript code is loaded from JavaScript files in the `wwwroot/index.html` file (Blazor WebAssembly) or `Pages/_Host.cshtml` file (Blazor Server) is important in some scenarios, such as JS interop. Add custom JS interop scripts after the `<script>` element for Blazor's script (`blazor.webassembly.js`/`blazor.server.js`). Add scripts before the closing `</body>` tag. For more information, see <xref:blazor/project-structure>.
```

Comment:
> Add custom JS interop scripts after the `<script>` element for Blazor's script (`blazor.webassembly.js`/`blazor.server.js`). Add scripts before the closing `</body>` tag.

This confused me a bit when reading. Would it be possible to show a code sample in the note?

```
<body>
   <script src="yourstuff.js"></script>
   <script src="blazor.webassembly.js"></script>
</body>
```

Reply:
In my experience, it was "Add custom JS interop scripts _before_ the `<script>` element for Blazor's script ..."